app/views/partials/_like_button.html.erb

```diff
@@ -0,0 +1,17 @@
+<% if liked %>
+  <%= button_to adopter_fosterer_liked_pet_path(id: LikedPet.find_by(adopter_foster_account_id: current_user.adopter_foster_account.id, pet_id: pet.id)&.id),
```

Comment:
Not usually a fan of doing queries in views. Not sure where this would best live, otherwise, either. If we move this method somewhere it's just going to be more code to achieve the same thing as we'd still need to pass the args to the method that then does the query...

Reply:
```suggestion
  <%= button_to adopter_fosterer_liked_pet_path(id: liked&.id),
```